index.html

```diff
@@ -211,5 +211,23 @@ <h1>About me</h1>
         </div>
       </div>
     </section>
+      <div class="contact-me">
+        <h1 class="contact-heading">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h1>
+        <form action="https://formspree.io/f/meqdprbd" class="contact-form" method="post">
+          <input type="text" class="contact-form-text" maxlength="30" placeholder="Full name" required>
+          <input type="email" class="contact-form-email" placeholder="Email address" required>
+          <textarea name="contact" class="contact-textarea" maxlength="500" placeholder="Enter your text here" required></textarea>
+          <div class="form-button"><input type="button" value="Get In Touch" class="contact-button"></div>

```

Comment:
- [ ] The form is not submitting properly due to the type of the input you added. Kindly change the type of the input button to `submit` so the form can be submitted.